automatic/jenkins/jenkins.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+    <metadata>
+        <id>jenkins</id>
+        <version>2.222.4</version>
```

Comment:
Is this the latest version available upstream? If it is, can you either change it to `0.0` or the latest version of Chocolatey Community Repository.